src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -27,6 +27,25 @@
     <Optimize>true</Optimize>
     <ErrorReport>prompt</ErrorReport>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug_Future|AnyCPU'">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Optimize>false</Optimize>
+    <ErrorReport>prompt</ErrorReport>
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
+    <Future>true</Future>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release_Future|AnyCPU'">
+    <Future>true</Future>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <Optimize>true</Optimize>
+    <ErrorReport>prompt</ErrorReport>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Future)' == 'true'">
```

Comment:
I was actually thinking we'd allow some sort of arbitrary identifier as a pivot to cross compile on.  I can imagine we might not want to use targetgroup since folks might need to cross-compile and have multiple targetgroups.

I don't like using assembly version since that leaks project-local state to the build.  Maybe some string that might coincidentally contain version, like BuildEdition or BuildVersion with values like Future or vNext or 1.3.

Agreed that this doesn't need to change for now.